pkg/syncer/broker/syncer.go

```diff
@@ -189,11 +188,6 @@ func (s *Syncer) Start(stopCh <-chan struct{}) error {
 	return nil
 }
 
-func (s *Syncer) GetBrokerFederatorFor(resourceType runtime.Object) federate.Federator {
-	f, found := s.federators[reflect.TypeOf(resourceType)]
-	if found {
-		return f
-	}
-
-	return nil
+func (s *Syncer) GetBrokerFederator() federate.Federator {
```

Comment:
This looks like a change that belongs to #86 I believe merge and rebase will take care of this, right?

Reply:
I based https://github.com/submariner-io/admiral/pull/95/commits/db1696ad5673e62afb375741ff475c9e5e98d602 on that commit to avoid conflict.